documentation/user_docs/docs/manual/Reshaping_etc.rst

```diff
@@ -1,196 +1,282 @@
-#############
-Reshaping etc
-#############
+#####################
+Other shape functions
+#####################
 
-replicate
-=========
+``replicate``
+=============
 
-This is a function to create a higher dimensional dataset by replicating along an additional axis. For example a d2d, which is essentially a matrix of intensity, can be replicated to make a d3d (an array of matrices) where the nth matrix is the same as the original one, or a d4d. In Matlab syntax this means e.g.
+This is a function to create a higher dimensional dataset by replicating along an additional axis. For example a d2d,
+which is essentially a matrix of intensity, can be replicated to make a d3d (an array of matrices) where the nth matrix
+is the same as the original one, or a d4d.
```

Comment:
Similarly d4ds are taken by further replicating the resulting d3d along the additional 4th dimension.

Noting also that you do not explicitly state that this is for dNds only, or could also be used for sqws - as you do at the end, but a preview here would be good.